ci/test-upgrade

```diff
@@ -47,6 +47,13 @@ function create {
     cd -
     cd $1/testgeomapfish
     git add --all
+    if [ $2 == '2.7' ]
```

Comment:
Why this custome case for 2.7 ?
Might be removed at some time ? Before 2.7 release ?
Should be add a # TODO: ? or a better test that we are on master ?

Reply:
+1 => `${MAJOR_VERSION}` thanks :-)